pkg/upgrade/upgrader/csp_upgrade.go

```diff
@@ -258,7 +258,8 @@ func (c *cstorCSPOptions) verifyCSPVersionReconcile(openebsNamespace string) err
 	statusObj.Phase = utask.StepErrored
 	// waiting for the current version to be equal to desired version
 	for c.cspObj.VersionDetails.Status.Current != upgradeVersion {
-		klog.Infof("Verifying the reconciliation of version for %s", c.cspObj.Name)
+		klog.Infof("Verifying the reconciliation of version for %s { phase:%s }",
```

Comment:
How about logging lastUpdated time ?

Reply:
Added last transition time to get the last phase change time.